docs/tutorials/control/scuttle-gamepad.md

```diff
@@ -20,73 +20,130 @@ date: "10 August 2022"
-<li>On the Component Config panel, leave <strong>Depends On</strong> set to empty.</li>
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "my-gamepad",
+      "type": "input_controller",
+      "model": "gamepad",
+      "attributes": {},
+      "depends_on": []
+    }
+```
+
+{{% /tab %}}
+{{< /tabs >}}
 
 The controller config adds the gamepad controller to your robot.
-However, it does not wire it up to any functionality.
-This requires a Service.
+However, it is not functional yet.
+To make it functional, you need to add the Base Remote Control Service.
 
-## Adding a Service
+## Add the Base Remote Control Service
 
-To link the controller's input to the base functionality, we need to add our first `service`.
-Services are the software packages that provide our robots with cool and powerful functionality.
+Services are software packages that provide robots with higher level functionality.
+To link the controller's input to the base functionality, you need to configure the [Base Remote Control Service](/services/base-rc/):
 
-1. Click **Create Service** under **services** at the top of the Viam app ([app.viam.com](https://app.viam.com)).
-2. Enter "Base Remote Control" for **Component** `type`. "Base Remote Control" is a service we provide for driving a rover with a gamepad.
-3. Enter **scuttle_gamepad** for the **Service** `name`.
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
 
-![Create service builder](/tutorials/scuttle-gamepad/pi-game-create-service.png)
+- Go to the **Services** subtab of your robot's **Config** tab.
+- In the **Create service** panel, click the **Type** dropdown and select `Base Remote Control`.
+- Enter `gamepad_service` for the **Service** **name**.
+- Click **Create service**.
 
-Copy and paste the following into the empty **Attributes** field:
-</OL>
+Copy and paste the following into the empty **Attributes** field, replacing `<your-base-name>` with your base's name.
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
- "base": "scuttle",
- "input_controller": "gamepad"
+ "base": "<your-base-name>",
+ "input_controller": "my-gamepad"
 }
 ```
 
-![Blank configuration json](/tutorials/scuttle-gamepad/pi-game-game-config-blank.png)
+<br>
+
+![Service configuration](/tutorials/scuttle-gamepad/gamepad-service-config.png)
+
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
+
+If instead of using the config builder, you prefer to write raw JSON, switch to **Raw JSON** mode on the **Config** tab.
+Add the following configuration for your base remote control service, replacing `<your-base-name>` with your base's name:
+
+```json {class="line-numbers linkable-line-numbers"}
+  "services": [
+    {
+      "name": "gamepad_service",
+      "type": "base_remote_control",
+      "attributes": {
+        "input_controller": "my-gamepad",
+        "base": "<your-base-name>"
+      }
+    }
+  ]
+```
```

Comment:
```suggestion
```

If you already have a `"services"` array with other services configured, add just the contents of the square brackets above to that array, rather than creating two different `"services"` arrays.
```